Add Fusion licensing content (#483)
seqeralabs/docs: fusion_docs/licensing.mdx

---
title: Fusion licensing
description: "Understand how Fusion licensing works"
date: "20 Feb 2025"
tags: [fusion, storage, compute, file system, license, quotas]
---

> **Note: Fusion licensing requirements and free tier quota will be enforced starting June 30th, 2025. Please [contact Seqera](https://seqera.io/contact-us) with any questions or licensing requests.**

Reviewer: ```suggestion
:::info
Fusion licensing requirements and free tier quotas will be enforced starting June 30th, 2025. [Contact Seqera](https://seqera.io/contact-us) with any questions or licensing requests.
:::
```

> A Fusion license allows you to use Fusion within a specific quota as measured by total throughput per month (defined by [Nextflow’s I/O metrics](https://www.nextflow.io/docs/latest/metrics.html#i-o-usage)).

Reviewer: ```suggestion
A Fusion license sets a specific quota for Fusion use, measured by total throughput per month (defined by [Nextflow’s I/O metrics](https://www.nextflow.io/docs/latest/metrics.html#i-o-usage)).
```

## For Seqera Cloud 

> - Seqera Platform will automatically add and manage a Fusion license on your behalf, up to the allowed quota of usage. Seqera Platform will track Fusion usage for all pipelines run within the Platform.

Reviewer: ```suggestion
- Seqera Platform will automatically add and manage a Fusion license on your behalf, up to the allowed usage quota. Seqera will track Fusion use for all pipelines run in Platform.
```

- The default quota for free use within Seqera Cloud is 100TB.

> - Once your quota is met/exceeded, you will be prevented from running additional Nextflow pipelines leveraging Fusion.

Reviewer: ```suggestion
- Once your quota is met or exceeded, you will be prevented from running additional Nextflow pipelines with Fusion.
```

## For Seqera Enterprise instances with connectivity to Seqera’s license server

> - Seqera Platform will automatically add and manage a Fusion license on your behalf, up to the allowed quota of usage. Seqera Platform will track Fusion usage for all pipelines run within the Platform.

Reviewer: ```suggestion
- Seqera Platform will automatically add and manage a Fusion license on your behalf, up to the allowed usage quota. Seqera will track Fusion use for all pipelines run in Platform.
```

- Once your quota is met or exceeded, you will be prevented from running additional Nextflow pipelines leveraging Fusion.

> ## For Seqera Enterprise instances without network connectivity and direct use with Nextflow  

Reviewer: ```suggestion
## For Seqera Enterprise instances without network connectivity, and direct Fusion use with Nextflow  
```

> [Contact Seqer](https://seqera.io/contact-us)a for a license that can be integrated into your workflow, or if you’d like to set up an evaluation.

Reviewer: ```suggestion
[Contact Seqera](https://seqera.io/contact-us) for a license that can be integrated into your workflow, or to set up an evaluation.
```
